timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -64,6 +79,29 @@ public void bringAllNodesOnline() {
     }
 
     @Test
+    public void blockedLockRequestThrows503OnLeaderElection() throws InterruptedException {
```

Comment:
any idea how long this takes to run? i'm worried it might be a little flaky given how hard it was for us to repro this.